Rakefile

```diff
@@ -6,12 +6,38 @@ Rake::TestTask.new do |t|
   t.pattern = './test/**/*_test.rb'
 end
 
-desc "update bundled certs"
+desc "Update bundled certs"
 task :update_certs do
-  require "restclient"
-  File.open(File.join(File.dirname(__FILE__), 'lib', 'data', 'ca-certificates.crt'), 'w') do |file|
-    resp = Faraday.get "https://raw.githubusercontent.com/bagder/ca-bundle/master/ca-bundle.crt"
-    abort("bad response when fetching bundle") unless resp.code == 200
-    file.write(resp.to_str)
+  require "faraday"
+
+  fetch_file "https://raw.githubusercontent.com/bagder/ca-bundle/master/ca-bundle.crt",
```

Comment:
FYI, the repository will be going away soon: https://github.com/bagder/ca-bundle#deprecated

We should probably grab the file directly from curl's site instead: https://curl.haxx.se/ca/cacert.pem